### ISSUE TITLE
Customising the FlowyElement

### ISSUE THREAD

**Issue by leungas** [Patcher56/vue-flowy#9]
Hi,

Loving this library!

Just wonder if there is a way to have custom element using vue-flowy. If so, where about is it possible to inject the HTML of specific HTML element can be done.

Thanks for the help!

Mark

**Comment by borger** [comment]
Indeed! Great work! To add a custom image or icon instead of the label, would be nice!
